l10n_br_account_banking_payment_bradesco_tributos/model/payment_mode.py

```diff
@@ -0,0 +1,63 @@
+# coding: utf-8
+# ###########################################################################
+#
+#    Author: Luis Felipe Mileo
+#            Luiz Felipe do Divino
+#    Copyright 2015 KMEE - www.kmee.com.br
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields
+
+
+class PaymentMode(models.Model):
+    _inherit = "payment.mode"
+
+    payment_order_type = fields.Selection(
+        selection_add=[
+            ('tax', u'Tributos'),
```

Comment:
This field is related:
https://github.com/OCA/bank-payment/blob/8.0/account_banking_payment_export/models/payment_mode.py#L73

It gives an error when trying to add new selection, should be removed.